packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -2053,7 +2053,7 @@ class PercentProperty extends DoubleProperty {
   /// The [showName] and [level] arguments must not be null.
   PercentProperty(
     String name,
-    double fraction, {
+    double? fraction, {
```

Comment:
needs to be nullable, otherwise what the heck is `ifNull` for? Also the super class double property has a nullable parameter here